notebook/tests/selenium/utils.py

```diff
@@ -222,6 +222,14 @@ def set_cell_metadata(self, index, key, value):
         JS = 'Jupyter.notebook.get_cell({}).metadata.{} = {}'.format(index, key, value)
         return self.browser.execute_script(JS)
 
+    def get_cells_mode(self):
```

Comment:
Ditto, I'm not sure it's clearer to have a method for this when it's only called once.

Reply:
Solved with the 3a0cb7b commit